app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/_order.less

```diff
@@ -33,6 +33,7 @@
 @order-create-icon-refresh__content: @icon-refresh__content;
 @order-create-icon-plus__content: @icon-plus__content;
 @order-create-icon-plus__hover__color: @color-very-dark-gray-black2;
+@order-address-info-width: 700px;
```

Comment:
Are there any existing variables to use for the order info width to make it's width relative to related areas if possible?